internal/service/device.go

```diff
@@ -212,3 +214,57 @@ func (h *ServiceHandler) GetRenderedDeviceSpec(ctx context.Context, request serv
 		return nil, err
 	}
 }
+
+// (PATCH /api/v1/devices/{name})
+func (h *ServiceHandler) PatchDevice(ctx context.Context, request server.PatchDeviceRequestObject) (server.PatchDeviceResponseObject, error) {
+	orgId := store.NullOrgId
+
+	currentObj, err := h.store.Device().Get(ctx, orgId, request.Name)
+	if err != nil {
+		switch err {
+		case flterrors.ErrResourceIsNil:
```

Comment:
do we need explicit cases here?
```suggestion
		case flterrors.ErrResourceIsNil, flterrors.ErrResourceNameIsNil:
```

Reply:
We dont actually. Thanks, I've updated the PR!